### PR TITLE
Php 7.3 compatibility

### PR DESCRIPTION
**Pull request by gignonje**
I just updated code to be compatible with PHP 7.3.

Seems to be working correctly, however the PHP 7.3 test seems to hang on Travis while it's working without issue on our machines.

Please comment if you see changes to be made.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "ext-mbstring": "*"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.8"
```
Comment (milesj) [inline composer.json]: If it's not too much work, maybe bump this to the latest?

File: src/Decoda/Decoda.php [hunk 2]
```diff
@@ -1264,7 +1264,7 @@ protected function _cleanChunks(array $chunks, array $wrapper = array()) {
                 case self::TAG_NONE:
                     // Disregard deeply nested text nodes if persist is disabled
                     if ($disallowed && !$parent['persistContent']) {
-                        continue;
+                        break;
```
Comment (milesj) [inline src/Decoda/Decoda.php]: These seems like breaking changes?

Comment (gignonje) [reply src/Decoda/Decoda.php]: According to the PHP documentation, `continue` in a `switch` context is like a `break`.
So this code should works exactly the same way as before.

See [this article](http://php.net/manual/en/migration73.incompatible.php) for more information.

File: composer.json (hunk 1 above)
Comment (gignonje) [reply composer.json]: This rule allows to install the latest release (PHPUnit 7.5.1) while maintaining ability to install a compatible version for PHP 5.X (PHPUnit 4.8). So the latest compatible version is going to be installed.

Comment (gignonje) [reply composer.json]: This rule allows to install the latest release (PHPUnit 7.5.1) while maintaining ability to install a compatible version for PHP 5.X (PHPUnit 4.8). So the latest compatible version is going to be installed.

File: src/Decoda/Decoda.php (hunk 2 above)
Comment (milesj) [reply src/Decoda/Decoda.php]: TIL, thanks for the info!

File: composer.json (hunk 1 above)
Comment (milesj) [reply composer.json]: 👍

**Comment by milesj** [comment]
Looks good. I'll test 7.3 stuff at home later today.

**Comment by gignonje** [comment]
Thank you for your answer. Please tell your results 😀

**Comment by gignonje** [comment]
Hi, thanks for merging this PR !

I saw that you updated version in README to 6.12.0, could you create a new Github release as well ?

Thank you

**Comment by milesj** [comment]
Weird, could of sworn I pushed tags. I'll push again when I get home.

**Comment by milesj** [comment]
@gignonje Thanks for the reminder. I pushed the tag, let me know if it still doesn't update.

**Comment by gignonje** [comment]
@milesj Thank you for pushing your tag, everything's good now !

**Pull request merged**